images/linux/scripts/installers/haskell.sh

```diff
@@ -43,6 +43,7 @@ echo "install cabal..."
 ghcup install cabal
 
 chmod -R 777 $GHCUP_INSTALL_BASE_PREFIX/.ghcup
+chown -R "$USER" $GHCUP_INSTALL_BASE_PREFIX/.ghcup
```

Comment:
`$USER` is the default user in the bare ubuntu VM in the github action, right?

Reply:
it is correct.